Source/JavaScriptCore/API/JSScriptSourceProvider.mm

```diff
@@ -24,6 +24,7 @@
  */
 
 #import "config.h"
+#import "JSBaseInternal.h"
```

Comment:
ditto

Reply:
This is needed to bring in the `JSC_OBJC_API_ENABLED` define, let me know if there's a better way.